bundler/lib/bundler/cli.rb

```diff
@@ -156,6 +156,7 @@ def self.handle_no_command_error(command, has_namespace = $thor_runner)
       dependency listed in the gemspec file to the newly created Gemfile.
     D
     method_option "gemspec", :type => :string, :banner => "Use the specified .gemspec to create the Gemfile"
+    method_option "gemfile", :type => :string, :banner => "Use the specified gemfile instead of Gemfile"
```

Comment:
I'd try to make it more clear that you're supposed to give this a _name_ for the Gemfile to be generated (that the command never expects a file that already exists).

```suggestion
    method_option "gemfile", :type => :string, :banner => "Use the specified name for the gemfile instead of 'Gemfile'"
```